src/Input/Textarea/Textarea.test.tsx

```diff
@@ -121,14 +123,19 @@ describe('<Textarea /> forwards ref to underlying textarea element', () => {
 
   test('ref is being forwarded correctly', () => {
     const ref = React.createRef<HTMLTextAreaElement>();
-    const { queryByLabelText } = render(<Textarea ref={ref} label={label} />);
+    const { queryByLabelText } = render(
+      <Textarea ref={ref} label={label} value={props.value} />
```

Comment:
Oh why we need to add `value` prop here? I think we should only add related props for unit test.

So you can add another test case for `value` prop. Can refer to https://github.com/glints-dev/glints-aries/pull/568/files#diff-dfe5d21ab8f9599028ecdf7e998d103507dd877e2ecffb0a3e403c598ea68bddR52

Reply:
Ohh its to cover the lines below that I highlighted and it's only will be triggered if we forward a ref and value to the Textarea comp.

Code in Textarea.tsx

```
  React.useLayoutEffect(() => {
    const textarea = textareaInputRef.current;

    if (!textarea) return;

    if (textarea.value.length > 0) { // this line of code
      setFloating(true);
    }

    setTextareaMaxHeight(~~(textarea.offsetHeight * 3.7 + 23 * 8 + 1));
  }, [textareaInputRef]);

```